tests/spark/test_spark_model_export.py

```diff
@@ -407,7 +408,7 @@ def test_log_model_calls_register_model(tmpdir, spark_model_iris):
                 model_uri, "AdsModel1", await_registration_for=DEFAULT_AWAIT_MAX_SLEEP_SECONDS
             )
     finally:
-        x = dfs_tmp_dir or sparkm.DFS_TMP
+        x = dfs_tmp_dir or MLFLOW_DFS_TMP
         shutil.rmtree(x)
```

Comment:
Looks like this `try-finally` is unnecessary because `dfs_tmp_dir` is created in `tmpdir` and `pytest` automatically removes `tmpdir`.

Reply:
This seems true. Will remove try-finally logic within this file.